### ISSUE TITLE
New version needed

### ISSUE THREAD

**Issue by isillo** [TinCanTech/easy-tls#316]
As per title. Old version pretty much doesnt work anymore with openvpn easy-rsa on windows.

**Comment by TinCanTech** [comment]
@isillo thanks for this reminder :+1:

**Comment by TinCanTech** [comment]
The tools which **Easy-RSA** ships for Windows are no longer a viable solution.

This will take some time to resolve and is not a priority.

You are advised to switch to Linux Sub-System for Windows or other Linux based solution.

**Comment by c-git** [comment]
Do you understand that this has been freely provided by the author? We have no claim to their time. If it's going to slowly for you then try to help. Create a PR to do the upgrade.

**Comment by TinCanTech** [comment]
Sorry for the delay, I'll sort this out soon.

**Comment by isillo** [comment]
> Sorry for the delay, I'll sort this out soon.

Thank for trying sort this out soon.

**Comment by isillo** [comment]
@TinCanTech hi, any update?

**Comment by isillo** [comment]
hello? Are there any alternatives that allow users to produce output like easy-tls on windows?

**Comment by TinCanTech** [comment]
> hello? Are there any alternatives that allow users to produce output like easy-tls on windows?

Not to my knowledge.

**Comment by isillo** [comment]
This app still does not work properly on Windows. Any news at all when it will finally work normally, without bending reality? Because of this I have been stuck for almost two years on a leaky VPN setup. No TLS. Wide open. At this point every hacker on the planet can probably steal all my money, empty my bank accounts, and laugh. Seriously - any updates? Will this ever be fixed?

**Comment by c-git** [comment]
Did you see the response higher up from the author indicating that the tool that this project used to use is not longer viable? Have you already tried using any of the suggestions provided by the author?

> The tools which **Easy-RSA** ships for Windows are no longer a viable solution.
> 
> This will take some time to resolve and is not a priority.
> 
> You are advised to switch to Linux Sub-System for Windows or other Linux based solution.

**Comment by isillo** [comment]
Sure, I’ve read the ideas above, but they are not optimal. I’m looking for a professional, straightforward solution - two minutes and done - without installing Linux or emulating it. I’m pushing towards changing the priority to the highest level, not looking for any workarounds, substitutes, or general advice.

**Comment by TinCanTech** [comment]
@isillo 

> This app still does not work properly on Windows. Any news at all when it will finally work normally, without bending reality? Because of this I have been stuck **for almost two years on a leaky VPN setup. No TLS. Wide open**. At this point every hacker on the planet can probably steal all my money, empty my bank accounts, and laugh. Seriously - any updates? Will this ever be fixed?

There is absolutely no way that you have an OpenVPN server without TLS.

If you do then you are the _elephant in the room_.

**Comment by TinCanTech** [comment]
@c-git 

> Did you see the response higher up from the author indicating that the tool that this project used to use is not longer viable? Have you already tried using any of the suggestions provided by the author?
> 
> > The tools which **Easy-RSA** ships for Windows are no longer a viable solution.
> > This will take some time to resolve and is not a priority.
> > You are advised to switch to Linux Sub-System for Windows or other Linux based solution.

Thank you very much.

**Edit**:
-- 

Because this is my repo, I am going to make this comment only once. 

Gender bullshit pisses me off and I will not accept ANY "gerderism".

I'm a **scruffy looking nerf** and I did not tell you my gender because that is none of your fucking business. And I'm a nerf not a nerf herder. **MOO**

**Comment by TinCanTech** [comment]
@isillo You can contact me directly by my email. Fees will apply.

**Comment by TinCanTech** [comment]
officially censored. Not by me by you-know-who ...
